### PR TITLE
Proper label string encoding

### PR DESCRIPTION
**Pull request by leo-p**

### CODE REVIEW

**Comment by alemoreau** [comment]
Care, unidecode behaves differently with python2

> The module exports a function that takes an Unicode object (Python 2.x) or string (Python 3.x)
https://pypi.org/project/Unidecode/

See tests for python2.7
`=============================== warnings summary ===============================
tests/test_draw.py::test_e2e_image_draw_image
  /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/unidecode/__init__.py:46: RuntimeWarning: Argument <type 'str'> is not an unicode object. Passing an encoded string will likely have unexpected results.
    _warn_if_not_unicode(string)`

**Comment by leo-p** [comment]
Removed unidecode for license reasons.

To do: Handle special characters Issue https://github.com/Deepomatic/deepocli/issues/58

File: deepomatic/cli/cmds/infer.py [hunk 1]
```diff
@@ -45,15 +46,16 @@ def __call__(self, frame):
...
             if self._draw_labels:
                 label = pred['label_name']
             if self._draw_labels and self._draw_scores:
                 label += ' '
             if self._draw_scores:
                 label += str(round(pred['score'], SCORE_DECIMAL_PRECISION))
+            label = unidecode(label)
             # Make sure labels are ascii
-            label = str(label.encode('ascii', 'replace'))
+            label = label.encode('ascii', 'replace').decode('ascii')
```
Comment (maingoh) [inline deepomatic/cli/cmds/infer.py]: you shouldn't need this line, the purpose of unidecode is to return ascii

Comment (maingoh) [reply deepomatic/cli/cmds/infer.py]: if you can also put a comment to explain why we need to unidecode (it is not supported by either opencv or the font)

**Pull request merged**